### PR TITLE
view detail content of post with click post in list

### PR DESCRIPTION
**Pull request by cjh5414**
- add Functional test for view post
- add unittest for view post
- add view_post.html
- add view_post in board/views.py
- view post url is 'posts/(\d+)/' % (post_id)

### CODE REVIEW

**Comment by cjh5414** [comment]
- add '목록' button that back to post_list on view_post

File: kboard/board/templates/post_list.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
     <h2>Post list</h2>
     <table id="id_post_list_table">
         {% for post in posts %}
-            <tr><td>{{ forloop.counter }}: {{ post.title }}</td></tr>
```
Comment (darjeeling) [inline kboard/board/templates/post_list.html]: in this case I think counter is not enough.
`post.id` is righ thing right?

File: kboard/board/templates/post_list.html [hunk 2]
```diff
@@ -8,7 +8,7 @@
...
+            <tr><td><a href="/posts/{{ forloop.counter }}">{{ forloop.counter }}: {{ post.title }}</a></td></tr>
```
Comment (darjeeling) [inline kboard/board/templates/post_list.html]: in this case I think counter is not enough.
`post.id` is righ thing right?

Comment (darjeeling) [reply kboard/board/templates/post_list.html]: and you can use `{{ url }}` in template
if you are using `url` in template tag, you don't need to hard code url after this time
https://docs.djangoproject.com/en/1.10/ref/templates/builtins/#url

**Review by darjeeling — CHANGES_REQUESTED** [review]

**Comment by cjh5414** [comment]
- change `forloop.counter` to `post_id` in post_list.html
- use `{{ url }}` in template
- change urlpatterns and url  for use `{{ url }}`

**Pull request merged**